database/core_templates/schema_diff.sql

```diff
@@ -0,0 +1,592 @@
+/***********************************************/
+/*** SCRIPT AUTHOR: conforma-server          ***/
+/***    CREATED ON: 2022-08-24T23:37:16.030Z ***/
+/***********************************************/
+
+--- BEGIN CREATE SEQUENCE "public"."user_application_join_id_seq" ---
+
+
+CREATE SEQUENCE IF NOT EXISTS "public"."user_application_join_id_seq" 
+	INCREMENT BY 1 
+	MINVALUE 1
+	MAXVALUE 2147483647
+	START WITH 1
+	CACHE 1
+	NO CYCLE;
+
```

Comment:
The core_templates should not have a schema_diff.sql, as the snapshot should have the exact same schema as the default sql init files. If you can't get rid of it and have it still work, let me know and I'll take a look at it -- I dealt with this problem a couple of weeks ago.

Reply:
I tried removing this and then regenarating the snapshot (exporting) it always have that. I think the init data isn't set correctly and that's what is causing it.